repository/kafka/src/main/java/com/expediagroup/streamplatform/streamregistry/repository/kafka/DefaultRepository.java

```diff
@@ -98,4 +99,14 @@ private void send(Event<SK, SS> event, List<CompletableFuture<Void>> futures) {
     //This is only used by ProducerBinding and ConsumerBinding
     throw new UnsupportedOperationException();
   }
+
+  @Override
+  public boolean delete(ME entity) {
+    List<CompletableFuture<Void>> futures = new ArrayList<>();
+    Entity<SK, SS> stateEntity = converter.convertEntity(entity);
+    StatusEntry status = stateEntity.getStatus().getEntries().get(0);
+    send(Event.status(stateEntity.getKey(), status), futures);
```

Comment:
I'm probably missing something here, but I'd have though this should be sending a `Event.specificationDeletion` rather than a status event.

I don't think this is actually going to delete anything on the backing kafka topic.

Reply:
updated to use spec deletion